examples/autorenderhint/src/index.js

```diff
@@ -0,0 +1,154 @@
+'use strict'
+
+const Prism = require('prismjs');
+const Video = require('twilio-video');
+const DataSeries = require('../../util/timelinegraph').DataSeries;
+const GraphView = require('../../util/timelinegraph').GraphView;
+const getSnippet = require('../../util/getsnippet');
+const getRoomCredentials = require('../../util/getroomcredentials');
+const helpers = require('./helpers');
+const joinRoom = helpers.joinRoom;
+
+const renderDimensionsOption = document.querySelector('select#renderDimensionsOption');
+const videoContainer = document.querySelector('div#video-container');
+const videoEl = document.querySelector('video#remotevideo');
+const visibilityToggleButton = document.querySelector('button#visibilityToggleButton');
+const trackIsSwitchedOff = document.querySelector('span#trackIsSwitchedOff');
+let roomP1 = null;
+let remoteVideoTrack = null;
+let startVideoBitrateGraph = null;
+let stopVideoBitrateGraph = null;
+
+/**
+ * Set up the bitrate graph for audio or video media.
+ */
+ function setupBitrateGraph(kind, containerId, canvasId) {
```

Comment:
is this used by manual render hint as well? Maybe we should move this to a https://github.com/twilio/video-quickstart-js/tree/master/examples/util

Reply:
I actually grabbed this graph off of the [bandwidthconstraints example](https://github.com/twilio/video-quickstart-js/blob/de923f101b5cd8493eb94a11f78f1fb8f97b32e8/examples/bandwidthconstraints/src/index.js#L130), you think I should move it there?